### DISCUSSION TITLE
auto bookmark - use the application without phone

### DISCUSSION THREAD

**Discussion by ifsergiy** [PaulWoitaschek/Voice#1881] [Ideas]
**use the application without phone**

use headset or headphones to add **bookmark**, or even simples — make the possibility in setting to make auto bookmark if pause media file

**Comment by dotancohen** [comment]
This is an interesting idea. Most bluetooth headsets have a button for e.g. answering and ending a call. Does the app have access to this button's state, to add a bookmark.
Perhaps there should be a configuration option, as I'm sure others would prefer for this button to pause/unpause instead.

**Comment by test2a** [comment]
I have an older phone, lenovo k8 which has a "music key" on the side. We could use this, say long press, double tap, single tap and such. Same goes for phones with google assistant button/bixby button
